### PR TITLE
Create Attendees Models and APIs

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: trainings/tests.py [hunk 1]
```diff
@@ -95,3 +94,84 @@ def test_api_create_event(self):
...
+            "event": event,
+            "participant": participant
+        }
+        attendee = Attendee.objects.create(**attendee_data)
+        # Execute
+        response = self.client.get(
+            '/api/v1/attendees/%s/' % attendee.id,
+            content_type='application/json')
+        # Check
+        self.assertEqual(response.data["survey_sent"], False)
```
Comment (imsickofmaps) [inline trainings/tests.py:145]: quite a limited set of checks?

File: trainings/tests.py [hunk 2]
```diff
@@ -95,3 +94,84 @@ def test_api_create_event(self):
...
+            "participant": "/api/v1/participants/%s/" % participant.id
+        }
+        # Execute
+        response = self.client.post('/api/v1/attendees/',
+                                    json.dumps(post_data),
+                                    content_type='application/json')
+        # Check
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        d = Attendee.objects.last()
+        self.assertEqual(d.survey_sent, False)
```
Comment (imsickofmaps) [inline trainings/tests.py:190]: Same, not much asserting going on

**Comment by imsickofmaps** [comment]
Just some more asserts then :+1:

**Pull request merged**